src/covid_model_seiir_pipeline/postprocessing/model/aggregators.py

```diff
@@ -0,0 +1,124 @@
+"""Postprocessing aggregation strategies."""
```

Comment:
Only new things here are aggregating all the way up a hierarchy instead of just national levels + global.  `git mv` can't always save us.

Reply:
There's some code duplication here that could stand to be factored out.  Will do it if I ever have to touch this again.